extensions/ql-vscode/src/vscode-tests/run-integration-tests.ts

```diff
@@ -94,3 +104,25 @@ async function main() {
 }
 
 main();
+
+
+function getLaunchArgs(dir: TestDir) {
```

Comment:
This was uglier than I had hoped. `'--disable-extensions'` needs to be called for minimal and no workspace runs otherwise ambient extensions get in the way of the run. This _may_ not be a problem on the build server, but it is definitely a problem locally.

Reply:
Annoying but fine for now, since I see you manually install the Test Explorer.